chase/frontend/app/participant/layout.tsx

```diff
@@ -0,0 +1,36 @@
+import "@/app/globals.scss";
+import { Inter } from "next/font/google";
+
+//theme
+import "primereact/resources/themes/lara-light-indigo/theme.css";
+//core
+import "primereact/resources/primereact.min.css";
+//icons
+import "primeicons/primeicons.css";
+
+
+const inter = Inter({ subsets: ["latin"] });
+
+export const metadata = {
+  title: "Chase",
+};
+
+export default function RootLayout({
+  children,
+}: {
+  children: React.ReactNode;
+}) {
+  return (
+    <html lang="de">
+      <body className={inter.className}>
+        <div className="flex h-screen w-screen bg-gray-light">
+          <div className="m-5 rounded-3xl bg-white flex shadow-md w-full overflow-hidden">
```

Comment:
Hmm. But then my design doesnt make any sense anymore. I can make the margins smaller, but I specifically designed it to look like a window, so when used in full screen, it mimics the feeling of working in an application.

If you wanted to ommit the margin alltogether, one would have to redesign the navbar as well, since it does not work with the rounded corners as well.

But I would suggest to keep this in mind and hear what first testers inside DMUN think of this and come back to it then, since it is a small thing to adapt, if requested.

But I totally get your point, its just that I ignored the thought to stick to my design guts :D

Reply:
On that note, what do you think of the idea to publish this as a PWA in Chrome as well?

Reply:
Thats probably a good way to go and as you said just a minor adjustment. Generally we should try to keep visual complexity always as low as possible.

I'm a big fan of PWAs but their support is limited. Safari and Firefox both pretty much prevent using them effectively. But publishing a manifest and letting the user decide is probably not harming anyone so yeah lets do that